iFixit/react-commerce
# Switch icons to Fontawesome

closes #668 

This PR moves the icon from Chakra to Fontawesome (we moved the icons of the whole project - not only the product page ones) 

## QA

1. Open the Vercel preview
2. Verify that no functionality is impacted 
3. Verify that the design is aligned with the mockup (there are just a few minor tweaks that we did in code only, but 99% should be as in the mockup)
